src/PowerPing/Ping.cs

```diff
@@ -373,16 +374,17 @@ private PingResults SendICMP(PingAttributes attrs)
                     IsRunning = true;
                 }
 
+                // Include sequence number in ping message
+                ushort sequenceNum = (ushort)index;
+                Buffer.BlockCopy(BitConverter.GetBytes(sequenceNum), 0, packet.message, 2, 2);
+
                 // Fill ICMP message field
                 if (attrs.RandomMsg) {
                     payload = Encoding.ASCII.GetBytes(Helper.RandomString());
                     Buffer.BlockCopy(payload, 0, packet.message, 4, payload.Length);
                 } else if (attrs.UsePingCookies) {
```

Comment:
I think we can just remove all the PingCookies stuff, seeing as your changes basically do what PingCookies were supposed to do, and better.

Reply:
I removed the PingCookies stuff.